ocaml/xapi/xapi_pgpu_helpers.ml

```diff
@@ -151,11 +151,16 @@ let assert_destination_pgpu_is_compatible_with_vm ~__context ~vm ~host ?remote (
     | None -> Db.PGPU.get_supported_VGPU_types ~__context ~self:pgpu
     | Some (rpc, session_id) -> XenAPI.PGPU.get_supported_VGPU_types rpc session_id pgpu
   in
-  let get_pgpus host =
+  let get_pgpus_of_host host =
```

Comment:
For consistency, I'd rather see the two `get_pgpus_...` function names have the same choice of `in` and `of`.